source/opt/debug_info_manager.cpp

```diff
@@ -146,6 +146,25 @@ void DebugInfoManager::RegisterDbgDeclare(uint32_t var_id,
   }
 }
 
+// Create new constant directly into global value area, bypassing the
+// Constant manager. This is used when the DefUse or Constant managers
+// are invalid and cannot be regenerated due the module being in an
```

Comment:
.. cannot be regenerated **due** the module being ..

--> .. cannot be regenerated **due to** the module being ..

?

Reply:
got it.